src/test/resources/mappings/ecs-tlr-request.json

```diff
@@ -0,0 +1,17 @@
+{
+  "mappings": [
+    {
+      "request": {
+        "method": "POST",
+        "url": "/tlr/ecs-tlr"
+      },
+      "response": {
+        "status": 200,
+        "body": "{\"id\": \"571b0a2c-8883-40b5-a449-d41fe6017083\",\"primaryRequestId\": \"571b0a2c-8883-40b5-a449-d41fe6017083\"}",
```

Comment:
I would avoid using a global stub for POST. Especially if it is returning such a minimal response